image.go

```diff
@@ -36,6 +36,11 @@ func deviceSpecifics(devName string, width, height int) (*gift.GIFT, error) {
 				gift.Rotate90(),
 				gift.FlipVertical(),
 			), nil
+		case "Streamdeck (original v2)":
+			return gift.New(
+				gift.Resize(width, height, gift.LanczosResampling),
+				gift.Rotate180(),
+		), nil
```

Comment:
Since the Streamdeck Original V2 and the Streamdeck XL have the same image settings, you could merge their checks into a single case statement:

```go
func deviceSpecifics(devName string, width, height int) (*gift.GIFT, error) {
	switch devName {
		case "Streamdeck XL", "Streamdeck (original v2)":
			return gift.New(
				gift.Resize(width, height, gift.LanczosResampling),
				gift.Rotate180(),
			), nil
		case "Streamdeck Mini":
			return gift.New(
				gift.Resize(width, height, gift.LanczosResampling),
				gift.Rotate90(),
				gift.FlipVertical(),
			), nil
		default:
			return nil, errors.New(fmt.Sprintf("Unsupported Device: %s", devName))
	}
}
```

Reply:
@dhollinger good point thank you. See latest commit for the change